uni/test/mocks/integration/src/schedule_page_test.mocks.dart

```diff
@@ -9,6 +9,7 @@ import 'dart:typed_data' as _i6;
 import 'dart:ui' as _i11;
 
 import 'package:flutter/material.dart' as _i10;
+import 'package:flutter/widgets.dart';
 import 'package:http/http.dart' as _i2;
```

Comment:
Did you manually edit the mock files? If you did, please refer to the [Project's README](https://github.com/NIAEFEUP/uni/tree/develop/uni ) on how you can regenerate the mock files